app/v1/hateoas/HateoasLinks.scala

```diff
@@ -33,6 +33,15 @@ trait HateoasLinks {
   private def bfLossChangeRequest(appConfig: AppConfig, nino: String, lossId: String): String =
     bfLossUri(appConfig, nino, lossId) + "/change-loss-amount"
 
+  private def lossClaimBaseUri(appConfig: AppConfig, nino: String): String =
```

Comment:
Can we rename `collectionUri` to `bfLossesBaseUri` and this to `lossClaimsBaseUri` for consistency and clarity that they are the collection resource uris?

Reply:
This is now done.